src/systemcmds/pwm/pwm.c

```diff
@@ -649,18 +671,26 @@ pwm_main(int argc, char *argv[])
 						}
 					}
 
-					warnx("User abort\n");
+					PX4_INFO("User abort\n");
 					return 0;
 				}
 			}
 
-			usleep(2000);
+			/* Delay longer than the max Oneshot duration */
+
+			usleep(2542);
```

Comment:
I need to check if we have enabled tick-less, but we do essentially have the ability for quite accurate usleep now.